pkg/cmd/run.go

```diff
@@ -127,6 +132,18 @@ doppler run --command "YOUR_COMMAND && YOUR_OTHER_COMMAND"`,
 				}
 			}
 
+			if addKey && preserveEnv {
+				// skip key if environment variables already contain this key
+				envKey := fmt.Sprintf("%s=", name)
+				for _, envVariable := range env {
+					if strings.HasPrefix(envVariable, envKey) {
+						utils.LogDebug(fmt.Sprintf("Ignoring Doppler secret %s", name))
```

Comment:
Should this be `envKey`?

Reply:
Given a doppler secret `API_KEY`, `secret` is `API_KEY` and `envKey` is `API_KEY=`